Giraffe.sln

```diff
@@ -37,6 +37,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "_build", "_build", "{8F5B79
 EndProject
 Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "Giraffe.Benchmarks", "tests\Giraffe.Benchmarks\Giraffe.Benchmarks.fsproj", "{2188E634-B828-4629-89B9-3680422460F5}"
 EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "EndpointRoutingApp", "samples\EndpointRoutingApp\EndpointRoutingApp\EndpointRoutingApp.fsproj", "{E4E1FB62-8268-4E0C-9F6E-25FF21E376E9}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "EndpointRoutingApp", "EndpointRoutingApp", "{C8D24EFD-B9A9-43DD-BE58-BB3C673869A5}"
```

Comment:
For some reason, Rider isn't happy about the way the nested projects are defined now. If I remove the endpoint routing solution dir and project and re-add it it's happy again 🤷‍♂️